package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jest-teamcity",
-  "version": "1.4.0",
+  "version": "2.0.0",
```

Comment:
and could I ask you to revert the version as well? I will dump the version when it's released

Reply:
ok